### PR TITLE
feat: Added Environment Variable ANONYMIZED_TELEMETRY=false

### PR DESCRIPTION
**Pull request by silentoplayz**
## Pull Request Checklist

- [x] **Description:** Briefly describe the changes in this pull request.
- [x] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [x] **Documentation:** Have you updated relevant documentation?
- [x] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?

---

## Description

Added: Environment Variable `ANONYMIZED_TELEMETRY=false` to `.env.example` & `Dockerfile` files to preload the `telemetry_user_id` file created by Chroma in Docker installation methods with a nil UUID of `00000000-0000-0000-0000-000000000000`.

---

### Added

Environment Variable `ANONYMIZED_TELEMETRY=false` to `.env.example` & `Dockerfile` files to preload the `telemetry_user_id` file created by Chroma in Docker installation methods with a nil UUID of `00000000-0000-0000-0000-000000000000`.

### Fixed

- `telemetry_user_id` file from creating a true anonymized telemetry UUID on users and nullifying the generated UUID out entirely.

### Changed

- [List any changes or updates]

### Removed

- [List any removed features or files]

Related #618

### CODE REVIEW

**Comment by Silentoplayz** [comment]
Will need someone to test this & ensure it is working please!

File: .env.example [hunk 1]
```diff
@@ -10,7 +10,8 @@ OPENAI_API_KEY=''
 # DO NOT TRACK
 SCARF_NO_ANALYTICS=true
 DO_NOT_TRACK=true
+ANONYMIZED_TELEMETRY=false
 
 # Use locally bundled version of the LiteLLM cost map json
 # to avoid repetitive startup connections
-LITELLM_LOCAL_MODEL_COST_MAP="True"
```
Comment (tjbck) [inline .env.example:16]: This should be set to "True" I believe.

Comment (tjbck) [reply .env.example:16]: https://github.com/BerriAI/litellm/blob/285a3733a968bbf0c94e76d0ff5d61a80e7429af/litellm/__init__.py#L232

**Comment by tjbck** [comment]
LGTM!

**Pull request merged**